doc/api/net.md

```diff
@@ -1125,6 +1129,8 @@ This property represents the state of the connection as a string.
 * If the stream is readable and writable, it is `open`.
 * If the stream is readable and not writable, it is `readOnly`.
 * If the stream is not readable and writable, it is `writeOnly`.
+* If the stream is not opened or is neither readable nor writable, it is
```

Comment:
Perhaps we should avoid using words like 'open' here because it's an actual `readyState` value and might be confusing?

Perhaps something like:
```suggestion
* If the stream is not connecting, connected, or is otherwise neither readable nor writable, it is
```